.gitignore

```diff
@@ -1,4 +1,5 @@
-out
+/out
 node_modules
 *.vsix
-*.todo
\ No newline at end of file
+*.todo
+package.nls.*.json
```

Comment:
Why are these ignored?

Reply:
Because they are only generated though .vsix packaging and not needed in the repo.
Language files and translation ar all maintained in the i18n folder.

That's why i added a "How the localization technically works" section in the initial posting.
I will write down some basic information of the loca handling, which i collected in the last days and in several readmes.